Simplenote/src/main/res/layout/fragment_confirmation.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:layout_width="72dp"
+        android:layout_height="72dp"
+        android:layout_gravity="center"
+        android:layout_marginBottom="16dp"
```

Comment:
np: Hard-coded dimension values. There is a `dimens.xml` file with the standard dimensions used across the app. Using those would ensure the padding and margins transfer nicely to larger devices like tablets.

Reply:
This is now done here: 00ec8ef3a5b0670f631f6b9e83bbede7649ea29b